spec/javascripts/controllers/container_live_dashboard/container_live_dashboard_controller_spec.js

```diff
@@ -9,8 +9,12 @@ describe('containerLiveDashboardController', function() {
 
   beforeEach(function() {
     var $window = {location: { pathname: '/ems_container/42' }};
+    var $element = function() {
+      this.find = function() { return "_ops" };
+    };
```

Comment:
the mock doesn't make sense.. you're calling `$element.find("#whatever").val()` .. soo.. `"_ops".val()`? ;)